# Calling Variants (SNV, SV, CNV), limitations from sequencing technologies

Speaker: Victor Guryev

## Reference genome

**1000 genomes project** (1kG) : low coverage whole genome and deep exome sequencing of 2,500 healthy individuals to discover 95% of variants at 1% frequency. [1] [2]

[1]: Small variants: The 1000 Genomes Project Consortium, 2015. [Nature 526:68-74](https://www.nature.com/articles/nature15393)
[2]: Structural variants: Sudmant et al, 2015. [Nature 526:75-81](https://www.nature.com/articles/nature15394)

## Technology

### NGS platforms

1. Illumina:
  * HiSeq 2500
  * NextSeq
  * NovaSeq
2. Thermo Fisher:
  * Ion Torrent
  * Ion Proton
3. PacBio
  * RS II
  * Sequel
4. Oxford Nanopore
  * MinION
  * PromethION
5. Combining platforms: Long low-quality reads + Short high-quality reads = Long high-quality reads

### Sequencing by synthesis

1. Reaction mixture
  * Primer and DNA template
  * DNA polymerase
  * ddNTPs with flourochromes
  * dNTPS
2. Primer elongation and chain termination
3. Capillary gel electrophoresis separation of DNA fragments
4. Laser detection of flourochromes and computational sequence analysis

### Sequencing quality measures

1. Formula: $Phred value = -10 \times \log{10}{(Error_rate)},\ where Error_rate = purity_of_the_signal.$
2. Encoding base quality values with ACII systems. (As the 032 in ACII is _space_ which may cause the confusions, the encoding starts from 033 "!" mark.)

Note: Not all sequencing system starts encoding from 033.

### Sequencing data file format

1. FASTA: Sanger sequencing data (2 files)
  * DNA sequence (.fasta)
  * non-encoding sequencing base quality (.qual)
2. FASTQ: A variant of FASTA format (1 file, 4 lines per read)
  * \@Sequence name (Flowcell ID/Instrument, Lane, Tile and position)
  * Sequence
  * \+ (empty or maybe followed by read name again)
  * Quality information (encoded base quality)

## Alianment

### Algorithm 

1. Smith-Waterman algorithm for local alignments
2. Dynamic programming, finds most optimal alignment
3. Basic Local alignment search tool (BLAST): BLASTing against big genomes **takes a lot of CPU time**.
4. Modifications for faster mapping:
  * MEGABLAST: 
    * A greedy algorithm for **highly similar sequences**.
    * concatenates query sequences before mapping.
  * Indexing with **Hash Tables**: SSAHA
  * Indexing Genome (BFAST, PASS)
  * Indexing Reads (MAQ, RMAP, SHRiMP)
  * Burrows-Wheeler transform (Bowtie, BWA)

### Aligner choice

| Platform            |           Application           |
| :-----------------: | :-----------------------------: |
| PacBio              |         BLASER, BWA mem         |
| Illumina            | BWA, Bowties 2, CLC-Bio, Stampy |
| Nonopore            |      LAST, BLAST, BWA mem       |
| Ion Torrent, Proton |              TMAP               |

### Mapping quality

Mapping quality is defined similarly as base quality, except:
* Calculate the _Error rate_ according to the mapping error rate and the number of locations where read is mapped to reference genome. (When multiple equally good mapping locations are possible, mappers randomly select one of them)
* No need to encode

### Alignment output format

1. Sample BLAST output
2. SAM format: Read ID, Flag, Chr, Position, MapQ, CIGAR, Chr2, Position2, TLen, Sequence, Quality, Aln.score, Ns, Mismatches, GapO, Gaps, Alignment
3. Binary version:
  * BAM(Binary SAM): compressed much smaller file, stores the same info as SAM.
  * Sorted BAM: reads are ordered by **chromosome and genomic positions**
  * BAI: Index of BAM. It is needed for fast access to any chromosomal location (like dictionary)
4. CRAM:
  * Compressing the data basing on reference genome.
  * Controlled loss of quality information. It summarize base quality into:
    * Low quality: 0 - 7;
    * Medium quality: 8 - 16;
    * High quality: 17 - 24

Note: All format 1-3 conversion is done via [SAMTools program](http://samtools.sourceforge.net)

## Variant calling

1. pileup the reference genome and all reads.
  * Pileup files:
    * Chr: chromosome
    * Pos: position
    * Ref: base in reference genome
    * Cov: coverage level at this base
    * Bases: legends to indicate the status of bases in each read which are mapped at this location. Legends and their meanings:
      * G Non-reference base (G) at plus strand
      * g Non-reference base (G) at minus strand
      * .(dot) Reference base at plus strand
      * ,(comma) Reference base at minus strand
      * ^F First base of read, read mapped with quality "F"
      * $ Last Base of read
      * .-2TG Next two bases (TG) are missing
      * ,+1a insertion of base 'A' after this
      * * gap in alignment (due to deletion)
    * Qualities: encoded quality scores (for each reads at this positio
2. Marking/removing the PCR duplicates
  * Principle: It considers the reads with the same _start_ and _end_ position on the reference genome are the sequencing error propagated in duplicates (caused by PCR amplification).
  * Tools: BAMutil, Picard

Command:

```{bash}
samtools mpileup -uf ref.fa sample_merged.bam | bcftools view -vcg - > output.vcf
```

3. Local realignment (around indels)
4. Base quality recalibrate
  * Quality scores are critical for all downstream analysis
  * Systematic biases are a major contributor to bad calls
5. Small variant calling (could detect up to 20-50 bp long variants, 2 methods used buy GATK)
  * **UnifiedGenotyper**: Call SNPs and indels separately by considering each variant locus independently.
  * **HaplotypeCaller**: Call SNPs, indels, and some SVs simultaneously by performing a local de-novo assambly. (More popular now)
    * Recover the artificial SNVs caused by indel
    * Distinguishing compound heterozygotes
6. Variant quality score recalibration (VQSR)
  * AC: Number of chromosomes carrying alt allele
  * AN: Total number of chromosomes
  * AF: Allele frequency
  * DP: Depth of coverage
  * QD: QUAL score over depth
  * AB: Allele balance of re/alt in hets
  * HRun: Length of longest contiguous homopolymer
  * MQ: RMS MAPQ of all reads
  * MQ0: Number of MAPQ o reads at locus
  * SB: Estimated strand bias score

7. Large variants calling 
  * Structure Variants (SV) classes:
    * Copy-number varianta
      * Deletion
      * Duplication
    * Copy-balance variants
      * Inversion
      * Translocation
  * Detection methods
    * Read depth analysis (RD)
      * Scope: Copy-number changes
      * Tools: CNV-Seq, CNVnator, SegSeq, DWAC-Seq
    * Discordant pairs (DP)
      * Scope: Copy-number and copy-neutral SV at resolution close to base-pair
      * Tools: Breakdancer, 123SV
    * Split-read mapping (SR)
      * Scope: Prediction of copy-number and copy-neutral SV at nucleotide resolution
      * Tools: Pindel, SRiC
    * Genome assembly (AS)
      * Scope: Various types of SVs including large inserts
      * Tools (de novo assemblers): SOAPdenovo, ABYSS, Allpaths-LG
    * Multi-method approaches to SV discovery
      * [PINDEL](http://gmt.genome.wustl.edu/packages/pindel/): Split-read mapping (very specific for short and mid-size variants)
      * [DELLY](https://github.com/dellytools/delly): Discordant read and split-read methods
      * [LUMPY-SV](https://github.com/arq5x/lumpy-sv): Multi-method tool
      * SUVIVOR, MetaSV, Parliament: creating consensus or multi-sample callset
      * Parliament2: run multiple tools (Breakdancer, BreakSeq, CNVnator, Delly, Lumpy, Manta) and create consensus call set (using SURVIVOR). It is available as docker container.
    * Complex variants
      * Gene retrotransposition

