Be careful when calling X-chromesome sequencing. It might not fit the standard pipeline.

Sampling 
* samples swaps will occur at every step
* Build in checks: female/male, SNV-assay + SNV calls from NGS, process every sample twice, check Mendelian inheritance

Systematic errors

mono-nucleotide

QC

Variants called per exome
duplicate rates
exome coverage
% mapped reads
% reads on target
transversion/transition rate
contamination other organisms
compare with all previous analysis

Pipeline
* new pipeline/component
first re-run standard controls and samples

Uneven coverage-CG%
* PCR
* sample preparation: gene panel / exome capture

Different tools

Mapping reads
* unmapped contigs
* alt mapping

Probabalistic mapping

Gene conversion, homologous sequences

Call vatiants in targeted regions is not accurate, because the capture may be not accurate

Reference genome is not complete, some genes are missed from it. Some small size reference genome might be more complete, like dotch 
