---
title: "CCle cell line radiosensitivity"
author: shuyu
date: Sep 21, 2018
output: word_document 
---
Article: A genetic basis for the variation in the vulnerability of cancer to DNA damage
Author: Brain D. Yard, Drew J. Adams, et al. | Jounal：Nature Communication | Published Date: 2016-4-25

# High-throughput profiling platform

## Data:

* Samples: 533 cancer cell lines (26 types) from CCLE _(Fig.1a)_
* Platform: 384-well plate | 25-225 cells per well | CellTiter-Glo 
* Treatment: $\gamma$-ray(0, 1, 2, 3, 4, 5, 6, 8, 10 Gy) 9 days after radiation
* Integral survival values:The _area under the curve (AUC)_ was estimated by _trapezoidal approximation_.
$$Survival\ value = \Sigma_{i = 1}^{8} {[f(X_1) + f(X_2)/2]\times \Delta X}$$
$$Sensitivity = Survival\ value \times (7 \div log_2 10)$$
0: completely sensitive; 7: completely resistant.

## Results:

1. Comparing with clonogenic assay (25 cell ines _Fig.1b_)

   * The proliferation index approximated clonogenic survival across individual dose, with the best approximations occurring **within the GI~50~**(_Fig.S2_)
2. Overview the integral survival in all cancers.(_Fig.1c_)
   * Integral survival significantly varied across and within lineages.(_Fig.1c_)
3. Comparing then integral survival within lineages or same organ.
   * The majority of lineages are **normal distributed**, except two, **colorectal** and **uterine** (The have higher proportion of resistant cell lines than predicted by normal distribution)(_Fig.1d_ and _Fig.S2_).

## conclusion: 

The high-throughput platform accurately profiles cell lines from multiple lineages for radiation response and reveals a mostly Gaussian distribution of radiation response within lineage.

# X-rays and DNA-damaging drugs.

## Data: 

   * cell response to drugs [CTD2](https://ocg.cancer.gov/programs/ctd2/data-portal) 
   * cancer cell lines (1107) 
   * 481 compounds (545)
   * 384-well(1536-well)
   * control: stautosporine or MG-132; DMSO
   * CellTiter-Glo
   * measure: areas under percent-viability score; 
    Fitting DMSO-normalized concentration-response curve using for parameter nonlinear regression model:
    $$f(x) = b + \frac{h}{1+e^{-(\frac{x-\alpha}{\beta})}}$$

## Results:

Correlation: X-rays response vs drugs response

 * The most positively correlated compound: **VX-680** $\rightarrow$ pan-inhibitor of the Aurora kinase $\rightarrow$ regulating **chromosome segregation** and **cytokinesis** during mitosis $\rightarrow$ leads to cell death by mitotic catastrophe.
 * **Genotoxic chemotherapeutics** (etoposide, paclitaxel, doxorubicin, bleomycin and others) likely to be correlated with radiation **sensitivity**.

## conclusion: 

There are similar genetic dependencies between genotoxic compounds and radiation.

# SCNA regulate survival after irradiation

## Data: 

* Affymetrix Genome-Wide Human SNP Array 6.0 Raw Affymetrix CEL files
* Processing:
    1. convert: SNP array Raw CEL files -> GenePattern pipeline & hg18 Affymetrix probe annotations -> a single value for each probe set representing a SNP allele or a copy number probe -> 
    2. normalization: prob set-specific linear calibration curves -> the most similar HapMap normal samples -> segmentation of normalization log2 ratios($log_2(CN/2)$):circular binary segmentation algorithm -> median centring of the segment values to a value of 0 in each sample.)-> 
    3. Quality check: 
        * visual inspection of array pseudo-images
        * probe-to-probe noise variation between copy number values
        * confidence levels of Birdseed genotyping calls
        * appropriate segmentation of the copynumber profile
    4. Identify focal regions of CNAs in individual samples and gene-level copy number (the maximum absolute-segmented value between the gene's genomic coordinates): GISTIC algorithm
    5. Separate binary variables representing amplifications (>0.7) and deletions (<0.7) were generated base on the GISTIC gene-level copynumber output.

## Results:

### Global SCNA vs radiation survival

**fSCNA** (The fraction of the genome that contains a SCNA) reflecting genomic instability. 
$$fSCNA = \frac{the\ length\ of\ segment\ with\ log_2{SCNA}> 0.2}{the\ length\ of\ all\ segments\ measured}$$

1. Global SCNA (fCNA) vs integral survival: _positively_ correlated (Pearson's r=0.27) (_Fig.2a_)
2. The number of mutation vs integral survival: _negatively_ correlated (Pearson's r=-0.19)
3. Two hypothesis:
   * The overall positive correlation of fSCNA with radiation survival could reﬂect **an increased capability** of tumour cells to **repair DNA double-strand breaks (DSB) after radiation**, utilizing mechanisms that are also used in the creation of SCNA such as non-homologous or micro-homology-mediated end joining or other error-prone repair mechanisms (for example, non-allelic homologous recombination).(create a _stochastic order_ of individual SCNA correlated with survival)
   * **Individual SCNA changed the expression of specific genes** within the structurally altered chromosomal segments. (identify _discrete SCNA_ on both sides of the survival spectrum)

### Individual SCNA vs survival (information co-efficient (IC) analysis)

1. IC correlation: Individual SCNA vs radiation response
   - The top 5 gene-level SCNA correlating with resistance and sensitivity(_Fig. 2b_)
   - The relative enrichment for discrete chromosomal segments that correlated with resistance or sensitivity suggested that individual SCNA events were **not randomly distributed** across the radiation response range
2. IC correlation: The expression of genes within the altered segments vs radiation survival and compared the means of the coefficients by pairwise analysis (resistant vs sensitive)(_Fig.2c_)
   - The amplified regions that correlated with radiation resistance had a significantly higher mean correlated coefficient than amplifications that correlated with radiation sensitivity. The inverse was observed for deleted regions that correlated with radiation resistance.**Amplified regions are more correlated with radiation sensitivity; deleted regions are likely to correlated with radiation resistance.**
   - In some case, changes in the expression of individual genes have previously been implicated in response to **cytotoxic stress**(E2F1, KIF3B, GSS, BCL2L1)

### correlation: SCNA vs survival (across lineages)

   1. The association between fSCNA and integral survival are **difference** in the degree and the direction across tumor lineages.(_Fig.2d_)
   2. Colorectal and uterine tumor cells presented a **correlation** between _radiation survival and SCNA_ and **anti-correlation** between _radiation survival_ and _mutation frequency_.(supported by previous researches "There is a subset of tumors in these two lineages presents low SCNA and high mutation frequency, inverse to others. It is attribute to either MLH1 silencing or and/or DNA polymerase $\varepsilon$ (POLE) mutation")(_Fig.2e_)
   3. The top 20 genes that correlated with radiation sensitivity in colorectal and uterine cancer, 6 and 8, respectively, have previously been associated with DNA repair.(_Fig.2f_)

## conclusion:

   1. SCNA regulate the response of cells to radiation-induced damage in part through **direct changes in gene expression**.
   2. There is an association between **low SCNA, high mutation frequency, DNA repair gene disruption and radiation sensitivity** in colorectal and uterine cancers.

# Gene mutations regulate cellular survival after irradiation

## Data

* mRNA gene expression data: GeneChip Human Genome U133 Plus 2.0 Array.
* Processing:
    1. convert: Raw Affymetrix CEL files $\rightarrow$ Robust Multi-array Average (RMA) $\rightarrow$ a single value for each probe set
    2. normalization: quantile normalization
    3. summarization: Either the original Affymetrix U133 + 2 CDF ﬁle or a [redefined custom CDF](http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/CDF_download.asp) ﬁle (ENTREZG—v15) are used

## Results

1. IC correlation: Identifying gene mutations vs radiation sensitivity (across all lineages)

- Higher IC values for genes with mutations and radiation sensitivity compared with resistance.(_Supplementary Data 5_)

- The top 19 genes that were associated with **radiation sensitivity** when mutated:

   - Seven of these genes have previously been implicated in the DNA damage response: TPR18, FLNA19, TP53BP1, SMG1, RANBP9, SMARCA4 and STAG3. (_Fig.3a_)

   - A subset of the 19 genes demonstrated **domain selectivity** in conferring sensitivity.(_Fig.S3_)

2. Determine whether identified genes were regulators of radiation response or merely associated with sensitivity.

  - (False correlation)Only associated: PIK3CA

    * Conflict:

      * PI3K/protein kinase B(AKT) pathway is frequently **deregulated** in human cancer.

      * Mutation in PIK3CA (activating PI3K/AKT) are associated with **radiation sensitivity.**

      * Activation of PI3K/AKT has previously been associated with **radiation resistance**.

    * Explanation:

      * Cell line with mutations in the p85-binding domain(p85 BD) are **sensitive** to radiation, driving the overall association(_Fig.3b_)

      * cell lines with mutations in the p85 BD of PI3KCA were largely from the uterine lineages (38% of the cell lines in Fig.3b), had a co-occurring mutation in PTEN(_Fig.3c_), and two representative cell lines were $\gamma$AKT (p-AKT) **replete**(_Fig.3d_)

      * => p85 BD mutations **retain** PI3K enzymatic activity.

      * Mutations in the PIK3CA p85 BD were also frequently identified (>20%)in uterine tumor samples profiled by TCGA network.(_Fig.3e_)

      * => PIK3CA p85 BD mutations reflect **markers of a radiosensitive lineage**(uterine carcinoma).

  - (True correlation) Mutations that conferred radiation resistance: KEAP1 and CUL3

    * KEAP1 is the key regulator of **oxidative stress response**. (ranked 9/1600 IC=0.112; P=0.0513)(_Supplementary Data 5)
    * (Previous) mutations in KEAP1 $\rightarrow$ **stabilization and activity** $\rightarrow$ Nrf2(encoded by NFE2L2) $\rightarrow$ radiation resistance. 
    * The highest frequency of KEAP1 mutations of any lineage profiled by the TCGA network to data(_Fig.3g_) is adenocarcinoma of the lung
    * The strongest association between KEAP1 mutation and radiation resistance was in LUAD cell.(IC = 0.352; p = 0.0224)
    * CUL3, encoding the ubiquitin ligase adaptor correlated with radiation resistance in adenocarcinoma of the lung.(_Fig.3h_)
    
* **Oxidative stress response**:

   * Sqstm1( $\leftrightarrow$ Keap1) $\rightarrow$ Nrf2(encoded by NFE2L2) $\rightarrow$ NQO1 and SQSTM1
   * CUL3(ubiquitin ligase adaptor $\leftrightarrow$ Keap1) $\nrightarrow$ Nrf2
    
## conclusion:

  Gene mutation determinants of **radiation-induce cellular damage response** and reveal distinct **functional consequences** for categories of mutations within individual genes.

# Gene expression profiles regulate survival after irradiation.

ssGSEA enrichment scores: 

   * Based on the weighted difference of the Empirical Cumulative Distribution Functions of the genes in the set relative to the genes not included in a individual set.
   * C2 sub-collection of the Molecular Signatures database(MSigDB)

## Results:

1. ssGSEA score vs integral survival(correlation)(_Fig.S4 and supplementary data 6_)
   * Radiation **sensitivity** correlated gene sets: DNA damage response, cell cycle, chromatin organization and DNA metabolism
   * Radiation **resistance** correlated gene sets: cellular signalling, lipid metabolism and transport, stem-cell state, cellular stress and inflammation.
   * **Suggests**: 
      1. The cellular processes **well-beyond DNA repair** regulate cellular survival after radiation.
      2. The active signalling pathways that correlate with radiation resistance reveals several targetable cellular receptors: TGF-$\beta$, EGFR, ER, NF$\kappa$B, JAK/STAT, AKT, FGFR, HER2, RAF, MEK, and Wnt/$\beta$-catenin.
2. 18,988 genes' expression vs integral survival values:
   * NQO1 and SQSTM1:(_Fig.4a and supplementary data 7_)
       1. NQO1(9th) and SQSTM1(11th) are highly correlated with **resistance**). They active in Oxidative stress response (consistent with the results in last section). 
       2. NQO1 and SQSTM1 gene expressions are **strongly correlated** across 979 CCLE cell lines.(_Fig.4b_). 
   * NFE2L2
       1. **NFE2L2 transcriptional activity** is associated with radiation survival across all lineages.(_Fig.4c_) and presents the highest overall activity in HCC and biliary cancer(_Fig.4d_)
       2. (Previous) Mutations in NFE2l2 and/or KEAP1 and/or deletions in CUL3 $\rightarrow$ Nrf2 activation (in lung squamous cancers)
 3. (Previous) SQSTM1 $\rightarrow$ Nrf2(in HCC)
 4. Integral survival values vs NFE2L2 activity and SQSTM1 expression:(_Fig.4e, f_): 
    * HCC had the strongest association between radiation survival and Nrf2 activity in any lineage profiled($R^2 = 0.41$ in HCC, $R^2 = 0.22$ in LUSC)
    * SQSTM1 expression was also correlated with the Nrf2 score.
 5. HCC patients (TCGA) with elevated SQSTM1 expression have a significantly lower overall survival(_Fig.4g_), indicating a poor overall prognosis for patients with active Nrf2 in HCC.
 6. The poor prognosis of NSCLC patients with active Nrf2.(_supplementary data 5_)

# Radiogenomic profiling of breast cancer

* An unbiased query of gene mutations or copy number changes that correlated with radiation survival in 28 breast cancer cell lines(_Supplementary Data 8_)
   * Top 50 segments correlating with resistance (_Fig.5a_)
      * Amplification of 17q12-22 (contain ERBB2) was associate with radiation **resistance**
      * Cell lines with amplification of 17q12-22 $\rightarrow$ ERBB2's CN and mRNA $\uparrow$ which correlated with radiation survival.(r = 0.43 and r = 0.45)
    * **suggests:** ERBB2 is the likely mediator of radiation resistance in the 17q12-22 amplicon.(with support from previous studies)

## AR regulates survival after irradiation in breast cancer

* ssGSEA projections (_Fig.5c_)
   * Multiple pathways were correlated with radiation resistance (ER, ERBB2, JAK/STAT3, and PI3K)
   * One of the most correlated signalling gene sets identified was associated with **androgen signalling**.
   * (Previous) AR inhibition increase the response to radiation in prostate cancer patients. 
* AR mRNA levels correlated with radiation survival (r = 0.48, _Fig.5d_)
* AR mRNA level in breast cancer is the 3rd rank among 28 tumor types. (_Fig.5e_)
* In TCGA samples, AR mRNA vs protein (r = 0.615), AR mRNA vs ESR1(ER$\alpha$) mRNA (r = 0.34) (Supplementary Fig.6)
* A subset of TCGA samples expressed AR and not ESR1.
* Some of the breast carcinoma cell lines expressed AR with varying levels of ER and ErbB2(_Fig.5f, S7_ )
* MDAMB453: AR+, ER-, ErbB2 inactive. $\rightarrow$ luminal AR-expressing subtype of triple-negative breast cancer.
* Dihydrotestosterone(DHT, 双氢睾酮) + MDAMB453: dose dependent rescue of cell growth.
* ENZ (competitor with DHT) + MDAMB453: decreases cell survival after irradiation.(_Fig.6a_)
* The protection and sensitization with DHT and ENZ only presents in AR+ cell lines (with or without ER+ and ErbB2 activity).(_Fig.6b_)

## AR protects breast cancer cells from DNA damage.

* neutral comet assay (DNA damage):(_Fig.6c_): DHT decrease and ENZ increase the 
* $\gamma$H2AX formation:(_Fig.6d_): support the result of comet assay
* **Conclusion**: Suppression of androgen signalling results in increased DNA damage and/or decreased repair in 
* (Previous) Androgen deprivation $\nrightarrow$ activity of PRKDC (DNAPKcs) $\dashrightarrow$ NHEJ
* $\gamma$DNAPKcs: high in Steroid-deprived media; low in DHT added media.
* **Suggests**: Androgen activity plays a pivotal role in the response of AR expressing breast cancer cells to ionizing radiation.
* In vivo experiments(_Fig.6f-i_): Supports the results above and indicates large dose of ENZ reduced the body weight of mouses during the treatment.
* **Suggests**: Selection of ENZ dose is likely to be important in the optimization of the therapeutic index.

# Conclusion:

Androgen abaltion cooperates with ionizing radiation to decrease tumor cell growth and survival in AR-positive breast adenocarcinomas.
